src/SFA.DAS.FindEpao.Domain.UnitTests/Courses/WhenConstructingACourseListItem.cs

```diff
@@ -58,7 +59,7 @@ public void Then_The_Empty_Constructor_Sets_The_Title_To_Unknown()
             var expectedString = "Unknown";
 
             //Act
-            var course = new CourseListItem(null, null, 0, false);
+            var course = new CourseListItem(null, null, 0, false, new string[] { "1.0" });
```

Comment:
Do you need a test for when the expected versions are null / empty like the other permutations?

Reply:
test added